package/com.unity.formats.usd/Samples/HelloUsd/HelloUsdExample.cs

```diff
@@ -29,8 +30,14 @@ class MyCustomData : SampleBase
 
         void Start()
         {
+#if UNITY_EDITOR
+            FocusConsoleWindow();
```

Comment:
`FocusConsoleWindow()` is a great addition, would be worth checking when the Console is not open. Presently appears to throw an exception and the sample may not continue?

Reply:
According to [this doc](https://docs.unity3d.com/ScriptReference/Debug-developerConsoleVisible.html) if the Console window is not open, any Debug.Log call will open it so it should be fine, but I did still implement a try catch just in case something goes catastrophically wrong